Kernel/FileSystem/FATFS/Inode.h

```diff
@@ -103,7 +104,7 @@ class FATInode final : public Inode {
     virtual ErrorOr<void> flush_metadata() override;
     virtual ErrorOr<void> update_timestamps(Optional<UnixDateTime> atime, Optional<UnixDateTime> ctime, Optional<UnixDateTime> mtime) override;
 
-    Vector<u32> m_cluster_list;
+    OwnPtr<Vector<u32>> m_cluster_list;
```

Comment:
An `OwnPtr` of `Vector` is a bit weird. This should probably just be wrapped in an `Optional` instead.